### PR TITLE
Connect api data

### PR DESCRIPTION
**Pull request by pddao**
- Add backend models according to api data 
- Display api data in frontend

### CODE REVIEW

**Review by Cathesso — APPROVED** [review]
Looks good to me :)

File: backend/src/main/java/de/pddao/backend/model/api/DwdApiResponse.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@Document(collection = "pollen-count") // zweites Repo nötig?
```
Comment (christophersiem) [inline backend/src/main/java/de/pddao/backend/model/api/DwdApiResponse.java:13]:
```suggestion
@Document(collection = "pollen-count") 
```

File: backend/src/main/java/de/pddao/backend/service/DwdApiService.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    @Autowired
+    public DwdApiService(RestTemplate restTemplate) {
+        this.restTemplate = restTemplate;
+    }
+
+    public List<DwdApiResponse> getPollenCount() {
+        ResponseEntity<DwdApiResponse> response = restTemplate.getForEntity(dwdApi, DwdApiResponse.class);
+
+        if (response.getBody() != null) {
+            System.out.println(response.getBody());
```
Comment (christophersiem) [inline backend/src/main/java/de/pddao/backend/service/DwdApiService.java:26]:
```suggestion

```

File: backend/src/test/java/de/pddao/backend/controller/PollenItemControllerTest.java [hunk 3]
```diff
@@ -33,6 +33,9 @@
     @Test
     @DisplayName("method listAllPollenItems() should return all pollenItems in repository")
     void listAllPollenItemsTest() {
+
+        //GIVEN
```
Comment (christophersiem) [inline backend/src/test/java/de/pddao/backend/controller/PollenItemControllerTest.java:37]:
```suggestion

```

File: frontend/src/hooks/usePollenCount.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  const [pollenCounts, setPollenCounts] = useState([]);
+
+  useEffect(() => {
+    loadPollenCount();
+  }, []);
+
+  const loadPollenCount = () => {
+    apiservice
+      .loadPollenCount()
+      .then((data) => setPollenCounts(data))
```
Comment (christophersiem) [inline frontend/src/hooks/usePollenCount.js:14]:
```suggestion
      .then(setPollenCounts)
```

File: frontend/src/navigation/BackButton.js [hunk 5]
```diff
@@ -1,13 +1,17 @@
...
 
   return (
-    <Wrapper onClick={() => history.goBack()}>
-      <img src={"/new-back-arrow.png"} alt="Back arrow" />
-    </Wrapper>
+    !homepage && (
+      <Wrapper onClick={() => history.goBack()}>
+        <img src={"/new-back-arrow.png"} alt="Back arrow" />
+      </Wrapper>
+    )
```
Comment (christophersiem) [inline frontend/src/navigation/BackButton.js:14]:
```suggestion
    homepage ?  <div></div> :
      <Wrapper onClick={() => history.goBack()}>
        <img src={"/new-back-arrow.png"} alt="Back arrow" />
      </Wrapper>
    ) 
```

**Review by christophersiem — APPROVED** [review]

**Pull request merged**